picard/ui/metadatabox.py

```diff
@@ -282,20 +283,32 @@ def edit(self, index, trigger, event):
                 return super().edit(index, trigger, event)
         return False
 
-    def event(self, e):
+    def keyPressEvent(self, event):
+        if event.matches(QtGui.QKeySequence.Copy):
+            self.copy_value()
+        elif event.matches(QtGui.QKeySequence.Paste):
+            self.paste_value()
+        else:
+            super().keyPressEvent(event)
+
+    def copy_value(self):
         item = self.currentItem()
-        if (item and e.type() == QtCore.QEvent.KeyPress and e.modifiers() == QtCore.Qt.ControlModifier):
+        if item:
             column = item.column()
             tag = self.tag_diff.tag_names[item.row()]
-            if e.key() == QtCore.Qt.Key_C:
-                if column == 1:
-                    self.clipboard = list(self.tag_diff.orig[tag])
-                elif column == 2:
-                    self.clipboard = list(self.tag_diff.new[tag])
-            elif e.key() == QtCore.Qt.Key_V and column == 2 and tag != "~length":
+            if column == 1:
+                self.clipboard = list(self.tag_diff.orig[tag])
+            elif column == 2:
+                self.clipboard = list(self.tag_diff.new[tag])
+
+    def paste_value(self):
+        item = self.currentItem()
+        if item:
+            column = item.column()
+            tag = self.tag_diff.tag_names[item.row()]
+            if column == 2 and tag != "~length":
```

Comment:
I think we should replace hardcoded column numbers by constants, and this tag match for length should prolly replaced by a function call (`can_be_pasted(tag)` or whatever), but that's unrelated to this PR.

Reply:
Actually I would like to see tags marked as readonly and checked for that. Then we could unify the checks here a bit.